packages/ckeditor5-mention/src/mentionui.js

```diff
@@ -170,6 +170,7 @@ export default class MentionUI extends Plugin {
 			this._mentionsConfigurations.set( marker, definition );
 		}
 
+		editor.on( 'change:isReadOnly', () => this._hideUIAndRemoveMarker() );
```

Comment:
It may look unimportant, but in other places in code where we listen for the `change:isReadOnly` event we never create listener on the `editor`, but on the particular plugin, like [here](https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-find-and-replace/src/findcommand.js#L40):

```js
this.listenTo( editor, 'change:isReadOnly', () => {
...
```

This implementation ensures that listener is bound to the plugin instead of the editor itself, so if e.g. plugin becomes turned off (maybe at some point we'll have dynamic plugin management) the listener will be dismissed. Such an approach also allows to have a bit cleaner listener lists (for both editor and plugin) and understand what depends on what.